apitools/base/py/encoding.py

```diff
@@ -328,12 +328,15 @@ def encode_message(self, message):
         if isinstance(message, messages.FieldList):
             return '[%s]' % (', '.join(self.encode_message(x)
                                        for x in message))
-        message_type = type(message)
-        if message_type in _CUSTOM_MESSAGE_CODECS:
+
+        # pylint: disable=unidiomatic-typecheck
```

Comment:
I actually did that at first, then the linter complained because the line was too long :/